README.md

```diff
@@ -2,6 +2,12 @@
 
 V-pipe is a workflow designed for clinical applications of next generation sequencing (NGS) data to viral pathogens. It produces a number of results in a curated format.
 
+[![Snakemake](https://img.shields.io/badge/snakemake-≥4.8.0-brightgreen.svg?style=plastic)](https://snakemake.bitbucket.io)
+[![bio.tools](https://img.shields.io/badge/bio-tools-orange.svg?style=plastic)](https://bio.tools/V-Pipe)
+[![omic.tools](https://img.shields.io/badge/omic-tools-blue.svg?style=plastic)](https://omictools.com/v-pipe-tool)
+[![License: MIT](https://img.shields.io/badge/License-Apache_2.0-yellow.svg?style=plastic)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
We use the Apache 2.0 licence, not the MIT License

Reply:
thanks for spotting my error